.safety-policy.yml

```diff
@@ -207,6 +207,13 @@ security:
             reason: jupyterlab  issue in version 4.03, requires < 4.0.11 python >= 3.8
         64588:
             reason: jupyterlab  issue in version 4.03, requires < 4.0.11 python >= 3.8
+        65212:
+            reason: Fixed cryptography version <=42.0.2 requires Python>=3.8
+        65182:
+            reason: Notebook; Issue affects ver >=7.0.0,<=7.0.6, requires Python>=3.8
```

Comment:
Let's comment on the version that fixes the issue, not on the affected versions. Also, let's state where we use the fixed version. Also, 7.0.6 is not affected, it fixes the issue.

Suggestion: "Fixed notebook version 7.0.6 requires Python>=3.8 and is used there".

Reply:
Done

Reply:
Agreed.  I misread it since they normally state it with "<".